### ISSUE TITLE
Could you please share your trained weights?

### ISSUE THREAD

**Issue by sjg918** [Alvin2112/PSIDNet#1]
Hi, thank you for sharing your great results.
I am training PSIDNet using your code base.
I am training PSIDNet using KITTI Odometry Dataset.
However, I keep experiencing learning failure around 10epochs.
In particular, I can see that the loss value of stage 3 increases sharply around 10epochs.
I think it may be because the normalization using batch normalization is not properly used throughout the network.
Or maybe I am doing something wrong.
In any case, this training is too painful for me.

Could you please share your trained weights?
I would like to use them to compare your results with mine (Haze removal method).

---- 
PSIDNet trained for about 10 epochs works well.
Below is a haze image and a image of haze removal using a trained PSIDNet.
![left_i0 03](https://github.com/user-attachments/assets/68c6245e-e8ff-4212-b4c4-87f3ff99df45)
![left_p0 03](https://github.com/user-attachments/assets/9df665da-a91f-4286-9e3b-04c1bce7a654)
However, according to my own evaluation results(Using around 4500 images), PSNR dB is only 30.
I think it is probably a problem caused by not being able to continue training for a long time.
Below is my training logs.
![image](https://github.com/user-attachments/assets/2322391c-905b-483b-bea7-2368e79acc31)
![image](https://github.com/user-attachments/assets/392afd10-49b8-4975-8f18-3414fdbaae95)
![image](https://github.com/user-attachments/assets/3861ec81-a719-408d-8b52-1a0e530becb8)

Thank you.

**Comment by Alvin2112** [comment]
Thank you very much for your attention to this work. I have graduated and left school, so I need some time to find the pre-trained model. If I find it, I will upload it to the GitHub homepage as soon as possible.
________________________________
发件人: jingyuSong ***@***.***>
发送时间: 2024年8月2日 14:16
收件人: Alvin2112/PSIDNet ***@***.***>
抄送: Subscribed ***@***.***>
主题: [Alvin2112/PSIDNet] Could you please share your trained weights? (Issue #1)


Hi, thank you for sharing your great results.
I am training PSIDNet using your code base.
I am training PSIDNet using KITTI Odometry Dataset.
However, I keep experiencing learning failure around 10epochs.
In particular, I can see that the loss value of stage 3 increases sharply around 10epochs.
I think it may be because the normalization using batch normalization is not properly used throughout the network.
Or maybe I am doing something wrong.
In any case, this training is too painful for me.

Could you please share your trained weights?
I would like to use them to compare your results with mine (Haze removal method).

________________________________

PSIDNet trained for about 10 epochs works well. (Compared to other SOTA Stereo-based haze removal methods, e.g. SRDNet, BidNet,)
Below is a haze image and a image of haze removal using a trained PSIDNet.
left_i0.03.png (view on web)<https://github.com/user-attachments/assets/68c6245e-e8ff-4212-b4c4-87f3ff99df45>
left_p0.03.png (view on web)<https://github.com/user-attachments/assets/9df665da-a91f-4286-9e3b-04c1bce7a654>
However, according to my own evaluation results(Using around 4500 images), PSNR dB is only 30.
I think it is probably a problem caused by not being able to continue training for a long time.
Below is my training log.
image.png (view on web)<https://github.com/user-attachments/assets/2322391c-905b-483b-bea7-2368e79acc31>
Thank you.

―

**Comment by ck324** [comment]
师兄，感谢您的高质量论文分享。
有三个问题想问一下，谢谢
1.代码中说测试使用test.py，是指的文件中的infer.py吗
2.可以分享一下代码的预训练权重吗
3.数据集后续会公开吗

谢谢您
